File: app/src/main/java/com/modori/kwonkiseokee/AUto/data/data/Exif.java
package com.modori.kwonkiseokee.AUto.data.data;

import com.google.gson.annotations.SerializedName;

public class Exif {
    @SerializedName("make")
    private String make;

    @SerializedName("model")
    private String model;

    @SerializedName("exposure_time")
    private String exposure_time;

    @SerializedName("iso")
    private String iso;
}

File: app/src/main/java/com/modori/kwonkiseokee/AUto/data/api/Exif.kt
package com.modori.kwonkiseokee.AUto.data.api

import com.google.gson.annotations.SerializedName

data class Exif(
        @SerializedName("make") val make: String,
        @SerializedName("model") val model: String,
        @SerializedName("exposure_time") val exposure_time: String,
        @SerializedName("iso") val iso: String)